pkg/kubectl/cmd/get/humanreadable_flags.go

```diff
@@ -90,13 +94,31 @@ func (f *HumanPrintFlags) ToPrinter(outputFormat string) (printers.ResourcePrint
 		columnLabels = *f.ColumnLabels
 	}
 
+	spec := *f.ExtraColumns
+	extraColumns := make([]printers.Column, len(spec))
+	if *f.ExtraColumns != nil {
+		// Break up user specified extra-columns JSON paths into relaxed JSON expressions
+		for ix := range spec {
+			colSpec := strings.Split(spec[ix], ":")
+			if len(colSpec) != 2 {
+				return nil, fmt.Errorf("unexpected extra-columns spec: %s, expected <header>:<json-path-expr>", spec[ix])
+			}
+			spec, err := RelaxedJSONPathExpression(colSpec[1])
```

Comment:
nit: since you're already using `spec` in the parent scope above, some potential confusion could be avoided here by choosing a slightly different name here

Reply:
Also, consider putting this loop in its own function in this same file